etc/gwenview.profile

```diff
@@ -14,13 +16,13 @@ caps.drop all
 nogroups
 nonewprivs
 noroot
-protocol unix
+protocol unix,netlink
 seccomp
-nosound
+tracelog
```

Comment:
don't know though if tracelog is a sensible default for a program like gwenview. Please correct if you feel like.